### PR TITLE
Jhorta tests

### PR DESCRIPTION
**Pull request by jhorta**
Wrote the primary tests for auth.js. Might need to add additional ones for company name and phone numbers.

### CODE REVIEW

**Comment by kkolli** [comment]
This looks great! Great job!

File: server/routes/auth/form.controller.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+'use strict';
```
Comment (kkolli) [inline server/routes/auth/form.controller.js]: Why is this under form.controller?

**Comment by kkolli** [comment]
@ncheema Actually this seems to have all your changes, we can just merge this, can you checkout this branch and verify that everything still works? Also under auth its says this is form.controller, can you change that to auth.controller? Then push to this, and we'll merge this instead of yours

**Comment by ncheema** [comment]
@kkolli it has the latest auth. but controller doesn't have the changes. i will pull from this and add the changes to the controller(also change the name)

**Comment by jhorta** [comment]
@kkolli @ncheema Alright, I'm not sure why my controller is outdated but I should have the latest changes from auth so just let me know what to do from here.

**Comment by kkolli** [comment]
Okay sweet, thanks guys!

**Comment by ncheema** [comment]
@kkolli @eddie4941 this has the latest changes(from #74, refactored, tests) so you guys can close #74 and we can go with this branch instead.

**Comment by kkolli** [comment]
Sweet this looks good! @eddie4941 Can you take a look and merge

**Comment by ncheema** [comment]
@eddie4941  wait on the merge. Let me test something first. thanks

**Comment by ncheema** [comment]
@eddie4941 @kkolli  All is well! please merge! thank you!

**Pull request merged**